development/webpack/README.md

```diff
@@ -0,0 +1,180 @@
+# MetaMask Development Build Tool
```

Comment:
This file is docs, but also a bit of a brain dump. Feel free to pick it apart, or ask for removals, additions, clarifications, etc.

Reply:
not a blocker, but maybe we could add a reference to this file in the main README, perhaps, [somewhere here](https://github.com/MetaMask/metamask-extension/blob/develop/README.md#development-builds)?

Reply:
done!